packages/itwinui-react/src/core/Buttons/Button/Button.tsx

```diff
@@ -27,6 +27,18 @@ export type ButtonProps = {
    * Icon shown after the main button content.
    */
   endIcon?: JSX.Element;
+  /**
+   * Passes props to the button label.
+   */
+  labelProps?: React.ComponentProps<'span'>;
```

Comment:
I think everything looks fine. Just a question. Why `span` here instead of something else?

Reply:
The label was already set as a `span` element in the code, so I wrote the prop accordingly. (See line 71 in the changes)